app/models/evss_claim_document.rb

```diff
@@ -23,6 +23,7 @@ class EVSSClaimDocument < Common::Base
 
   # rubocop:disable Layout/LineLength
   DOCUMENT_TYPES = {
+    '1489' => 'Hypertension Rapid Ready For Decision Summary PDF',
```

Comment:
As @tadhg-ohiggins mentioned, 
This new document code doesn't have a leading `L` and we got confirmation from VBMS that this was correct.